src/app/components/PriceInfo.tsx

```diff
@@ -3,63 +3,83 @@ import { useAppSelector } from "../hooks";
 
 export function PriceInfo() {
   const priceInfo = useAppSelector((state) => state.priceInfo);
-  const lastPrice = priceInfo.lastPrice;
-  const change = priceInfo.change24h;
-  const high = priceInfo.high24h;
-  const low = priceInfo.low24h;
-  const volume = priceInfo.value24h;
+  const lastPrice = priceInfo.lastPrice.toFixed(3);
+  const change = priceInfo.change24h.toFixed(2);
+  const high = priceInfo.high24h.toFixed(3);
+  const low = priceInfo.low24h.toFixed(3);
+  const volume = priceInfo.value24h.toFixed(3);
   const isNegativeOrZero = priceInfo.isNegativeOrZero;
+  const basePair = "XRD";
 
   return (
     <div className="flex justify-between py-2">
       <div className="flex flex-col items-start pl-8">
-        <span className="text-sm">Price</span>
+        <span className="text-sm font-bold text-secondary-content uppercase">
+          Price
+        </span>
         <span
           className={
-            isNegativeOrZero ? "text-xs text-red-500" : "text-xs text-green-500"
+            isNegativeOrZero
+              ? "text-sm font-bold text-red-500"
+              : "text-sm font-bold text-green-500"
           }
         >
           {lastPrice}
+          <span className="text-sm font-bold text-secondary-content">
+            {" "}
+            {basePair}
+          </span>
         </span>
       </div>
       <div className="flex flex-col items-start">
-        <span className="text-sm">24h Change</span>
+        <span className="text-xs font-bold text-secondary-content uppercase pb-1">
+          24h Change
+        </span>
         <span
           className={
-            isNegativeOrZero ? "text-xs text-red-500" : "text-xs text-green-500"
+            isNegativeOrZero
+              ? "text-sm font-bold text-red-500"
+              : "text-sm font-bold text-green-500"
```

Comment:
Instead of `text-red-500` and `text-green-500` please use colors from the theme (they are specified in `tailwind.config.js`, I believe these should be `text-error` or `text-success` (but maybe `text-error-content` or `text-success-content will look better`... I don't know, please check.

I would even go with what [Vlad defined in figma](https://www.figma.com/file/qushS15h87wjpoXaY6ND4W/DeXter-UI---refresh?node-id=501%3A1107&mode=dev) and dropped the conditional `isNegativeOrZero` coloring (and removed the variable from the state at all) and color only the number in `24HRS CHANGE` column with accent color (`text-accent`).

Reply:
text-error and text-success looked better when I test vs text-error-content text-success-content